### PR TITLE
YD-128 Run the goal file loader as part of the Docker image creation

### PR DESCRIPTION
**Pull request by Bert-R**

### CODE REVIEW

**Comment by Bert-R** [comment]
Don't pull this yet. The Dockerfile needs a change.

**Comment by Bert-R** [comment]
I made the change in the Dockerfile. If there's someone who can review and pull this, that would be great. If not, I'll do it myself later today.

**Comment by pgussow** [comment]
I will take a look

File: dbinit/src/main/docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM yonadev/hsqldb:2.3.2
+
+ADD DatabaseInitializer.jar DatabaseInitializer.jar
+ADD data data
+
+RUN ["java","-jar","DatabaseInitializer.jar","--spring.datasource.url=jdbc:hsqldb:file:thedatabase"]
```
Comment (pgussow) [inline dbinit/src/main/docker/Dockerfile]: Shouldn't the connection string be externalized? Because once MariaDB is in place this will change again.

Comment (Bert-R) [inline dbinit/src/main/docker/Dockerfile]: For MariaDB we'll anyway change this. Note that this image is derived from the HSQLDB image.

**Pull request merged**